java/pgjdbc/build.gradle.kts

```diff
@@ -17,13 +17,14 @@ repositories {
 }
 
 dependencies {
+    implementation("org.postgresql:postgresql:42.7.8")
```

Comment:
Doesn't the jdbc connector include pgJDBC as a transitive dependency? Or are we overriding the included driver version here?

Reply:
The logging library was also not used by the example at all and is pulled in as a transitive dependency. Removed both dependencies and updated the Maven build command. The jar packaging approach which it used previously doesn't work with JDBC as a transitive dependency but I think that's fine since that's not how you would package a project typically anyway.